app/components/Button.js

```diff
@@ -36,7 +36,10 @@ class Button extends React.Component {
             buttonStyle ? buttonStyle : styles.container,
             { height: buttonHeight },
           ]}
-          onPress={onPress}>
+          onPress={onPress}
+          accessible
```

Comment:
is `accessible` needed if the label is provided?

Reply:
from a merge